File: Data Structures & Algorithms/counting-bits/submission-0.js
class Solution {
    /**
     * @param {number} n
     * @return {number[]}
     */
    getCount(n){
        let count=0
        for(let i=0;i<10;i++){
            if(n&1) count++
            n=n>>1
        }
        return count
    }
    countBits(n) {
        let result =[]
        for(let i=0;i<=n;i++){
            result.push(this.getCount(i))
        }
        return result
    }
}

File: Data Structures & Algorithms/longest-common-prefix/submission-0.py
class Solution:
    def longestCommonPrefix(self, strs: List[str]) -> str:
        result=0
        
        for i in range(len(strs[0])):
            ch,count=strs[0][i],0
            for str in strs:
                if len(str)<=i or str[i]!=ch:
                    break
                count+=1
            if count==len(strs):
                result+=1
            else:
                break
        return strs[0][0:result]